python/sparktk/models/clustering/gmm.py

```diff
@@ -259,8 +259,9 @@ def cluster_sizes(self, frame):
 
     def predict(self, frame, columns=None):
         """method to predict on a given frame"""
```

Comment:
This doesn't have the proper function doc header with parameters/return value, etc.  (This comment goes for all predict() methods - a bunch of them don't have proper documentation)